src/fides/api/api/v1/endpoints/generic_overrides.py

```diff
@@ -165,6 +183,13 @@ async def list_dataset_paginated(
         filter_params=filter_params,
     )
 
+    # If applicable, filter by connection type
+    if connection_type:
+        filtered_query = filtered_query.where(
+            CtlDataset.fides_meta["namespace"]["connection_type"].as_string()
+            == connection_type.value
+        )
```

Comment:
I don't think this is a reliable query. Only some datasets have a namespace and even if they do, the connection_type might not be present. Can you remind me why we need to retrieve datasets by connection type?